src/components/post/PriceOfferCard/index.tsx

```diff
@@ -1,102 +1,188 @@
-import { Divider, ToggleButton, SelectBox, Text } from '@offer-ui/react'
+import { Divider, SelectBox, Text, Icon } from '@offer-ui/react'
 import type { ReactElement } from 'react'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import { Styled } from './styled'
 import type { PriceOfferCardProps } from './types'
+import { PriceOfferModal } from '../PriceOfferModal'
+import type { OfferForm } from '../PriceOfferModal/types'
 import { UserProfile } from '../UserProfile'
+import { useUpdateLikeStatusMutation } from '@apis/like'
+import { useGetPostOffersQuery, useCreateOfferMutation } from '@apis/offer'
+import { useGetPostQuery } from '@apis/post'
+import { SORT_OPTIONS } from '@constants/app'
+import { getTimeDiffText, toLocaleCurrency } from '@utils/format'
+import { useModal } from '@hooks'
+import type { SortOption, SortOptionCodes } from '@types'
 
 const PriceOfferCard = ({
-  offerList,
-  likeCount,
-  isLike
+  postId,
+  isSeller
 }: PriceOfferCardProps): ReactElement => {
-  const hasOffer = offerList.length > 0
-  const [isLikeToggle, setIsLikeToggle] = useState<boolean>(isLike)
+  const [sortOption, setSortOption] = useState<SortOptionCodes>(
+    SORT_OPTIONS[0].code
+  )
+  const {
+    isOpen: isOfferModalOpen,
+    openModal: openOfferModal,
+    closeModal: closeOfferModal
+  } = useModal()
```

Comment:
```suggestion
  const offerModal = useModal()
```
네이밍 변경 대신 `offerModal.isOpen`, `offerModal.openModal` 처럼 사용해도 괜찮을 것 같습니다!

Reply:
호옹 요렇게 쓰는게 더 간닫ㄴ하네요 ! 
이렇게 사용하도록 맞추면 modal키워드가 중복되니 close, open으로 변경하는 것도 좋을 것 같습니다

Reply:
[완료](https://github.com/price-offer/offer-fe/pull/203/commits/f4418950e2c876977d9d0097c520cd0746d2f0f0#diff-99868c718a54a36948d73a8b379e24547ec16c2d4007dd4b2675768b9b540c16R27)